library/nxos_switchport

```diff
@@ -72,17 +72,31 @@ options:
         aliases: []
     trunk_vlans:
         description:
-            - if mode=trunk, used as the vlan range to carry over trunk
+            - if mode=trunk and state=present:
+                - add vlans to trunk
+            - if mode=trunk and state=absent:
+                - remove vlans from trunk
+              
         required: false
         default: '1-4094'
         choices: []
         aliases: []
     state:
         description:
```

Comment:
Same thing here.  Please re-word (but logic is fine).  I also don't think we need to call out every example type.  Examples will help show this too.

When state=present, ensure the configuration for the give mode exists on the switch.  When state=absent, the access_vlan will be unconfigured when mode=access, and when trunk_vlans is used with state=absent, the given vlan(s) will be removed from the trunk.  State=default does not default the interface or mode, but ensure it's a an access port vlan 1 configured or trunk that carries all VLANs with 1 as the native vlan.

Note: Can't have any colons in the docstring.